package.json

```diff
@@ -68,6 +68,7 @@
     "eslint-plugin-import": "2.25.4",
     "ganache-cli": "^6.12.2",
     "husky": "^4.3.8",
+    "keyv":"4.5.0",
```

Comment:
Was this added accidentially?

Reply:
This was added to resolve this error - https://github.com/energywebfoundation/ew-credentials/actions/runs/3369032554/jobs/5588238008#step:6:800

Reply:
Hmm, ok. There is also https://www.npmjs.com/package/@types/keyv

Reply:
I saw this as well, but it is deprecated so I thought not to use it.